added blog post section to index

SingingApple opened the pull request:
Blog post fetches data from a dummy JSON file using blog.js file. Blogs currently display truncated article body, article title, article source, author's name and published date in a card on the landing page. For date formatting, momentjs has been used. 
Two .png files have also been added for logos under /img/icons/blog. (medium and hackernoon)

SingingApple committed:
Added blog post

rajat2502 commented:
@krrohitch Sir please review the design of the blog section.

rajat2502 commented:
> momentjs

Don't use any external libraries until it is too necessary.

SingingApple committed:
Card-img blog

SingingApple commented:
Added img to each card and aligned all items. Please review

rajat2502 commented on index.html:
```diff
@@ -2,231 +2,417 @@
+        </div>
+      </section>
+    </div>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.26.0/moment.min.js"></script>
```
Remove this line.

rajat2502 commented on js/Blog.json:
```diff
@@ -0,0 +1,31 @@
+      "published_on": "20 November 2019",
+      "source": "medium",
+      "link": "https://medium.com",
+      "image": "../img/zealicon.png"
```
For now, this is fine but later we will have hosted image links here

rajat2502 replied:
Also, don't save the blog website image in the images folder we can also add their link here.

SingingApple replied:
yes sir, links will also work as [image] value is being rendered for background URL.

rajat2502 commented on js/blog.js:
```diff
@@ -0,0 +1,70 @@
+//Async request to JSON file for Blog post
+const getPosts = async () => {
+  const response = await fetch("../js/Blog.json");
+  const data = response.json();
+  return data;
+};
```
We don't need to make an API call to the blog's file. You can directly import the object from the .json file.

SingingApple committed:
Fixed font and links

SingingApple commented:
- Changed card body font to Avenir regular
- Added hyperlink to card container
- changed background-size for card image
- decreased y-offset for box-shadow of cards
- changed fetch API 
@krrohitch @rajat2502  please review

rajat2502 commented:
@rex12543 Please review it design-wise.
https://deploy-preview-38--trusting-mayer-c6cce8.netlify.app

SingingApple committed:
Merge branch 'master' into master

SingingApple committed:
Merge branch 'master' into master

SingingApple committed:
resolved issues from merge conflict

SingingApple committed:
fixed smooth scroll

rex12543 commented:
> @rex12543 Please review it design-wise.
> https://deploy-preview-38--trusting-mayer-c6cce8.netlify.app

The blog cards can be scaled down in size.
Rest looks fine to me for now. ✌

rajat2502 commented:
@SingingApple, update this PR according to the new design. Also, make sure that you resolve any merge conflicts.

SingingApple committed:
new-blog-UI

SingingApple commented:
@rajat2502 please review!

SingingApple committed:
new-UI-2

SingingApple committed:
Merge branch 'master' into master

SingingApple committed:
Fonts and Med Logo
